VisibilityXds, SpectrumXds: xradio_version => creator

FedeMPouzols opened the pull request:
Resolves #332.

---
Note well:
```
This contribution is made under the current ALMA software agreements. 
(c) European Southern Observatory, 2024, 2025
Copyright by ESO (in the framework of the ALMA collaboration)
```

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/casangi/xradio?pullRequest=370) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/casangi/xradio?pullRequest=370) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/casangi/xradio?pullRequest=370) it.</sub>

kettenis commented:
The creator/version bits look good to me.

I'm not an expert on the WVR stuff, but isn't that more like a spectrum instead of visibility data?

FedeMPouzols commented:
> I'm not an expert on the WVR stuff, but isn't that more like a spectrum instead of visibility data?
I think so. For this type of MSv4 we have PROCESSOR/TYPE = "RADIOMETER" instead of "CORRELATOR".

When fixing some issues with the "wvr" type that made me think perhaps we should use a more generic "radiometer" xds type rather than the particular case of "wvr". I believe "wvr" is the only such non-correlated data that we have come across (simply because ALMA imports those WVR SPWs in its data processing), but there must be others out there.

These "wvr" are an exception in the sense that the data is not "correlated" but we are forced to put them in a "correlated_xds".
The "spectrum" type of correlated_xds I'd guess is reserved for proper correlated radio data spectrum.


Note: I just realized that the `if` that checks if it's a wvr MSv4 should also check the SPW name in addition to the intent. Perhaps we should just check if the PROCESSOR/TYPE is not CORRELATOR. That sounds more robust than relying on scan intents and SPW names, which have not always been duly populated (at least in early ALMA cycles).

FedeMPouzols commented:
Merging for now. @Jan-Willem : note that when deciding whether the type is "visibility" or "wvr", I replaced the check for scan intents contents with a check for `processor_info.type == RADIOMETER`. I also changed the "wvr" to a more generic "radiometer". That sounded more robust to me. Now the proper SPWs are being identified as "radiometer". I think so far we have only seen this type of non-correlated data in the ALMA "wvr"?
